### PR TITLE
Add AllCatchR container

### PR DESCRIPTION
**Pull request by NadineWolgast**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x ] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [x] LABEL about.documentation   
  - [x] LABEL about.license_file
  - [x] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

File: allcatchr/1.0.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    container="allcatchr" \
+    about.summary="The gene expression classifier ALLCatchR identifies B-precursor ALL subtypes and underlying developmental trajectories" \
+    about.home="https://github.com/ThomasBeder/ALLCatchR" \
+    software.version="1.0.0" \
+    version="1" \
+    extra.identifiers.biotools="allcatchr" \
+    about.copyright="2022, Thomas Beder" \
+    about.license="MIT" \
+    about.license_file="https://github.com/ThomasBeder/ALLCatchR/blob/main/LICENSE.txt" \
+    maintainer="Nadine Wolgast <NadineWolgast@gmx.de.br>"\
```
Comment (osallou) [inline allcatchr/1.0.0/Dockerfile]: invalid extra slash on last line, docker build fails

**Review by osallou — CHANGES_REQUESTED** [review]

File: allcatchr/1.0.0/Dockerfile (hunk 1 above)
Comment (NadineWolgast) [reply allcatchr/1.0.0/Dockerfile]: I'm not sure if the installation RUN statement is correct. It is a R package that needs devtools to be installed. Could you suggest me how I can change it so it works?

Comment (osallou) [reply allcatchr/1.0.0/Dockerfile]: I do not know how R works... so no idea

**Comment by osallou** [comment]
dockerfile failed to build.... would be nice to test locally the build before pushing updates

**Comment by osallou** [comment]
@NadineKraft you define

LABEL extra.identifiers.biotools="allcatchr"


but no allcatchr entry could be found in bio.tools , remove this line if not defined in biotools, or, better, add it in biotools before ;-)

**Pull request merged**